content/resources/tutorials/kafka-topics-tutorial.md

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
I'm not sure about the content of the tutorial. What kind of information are useful to users? Should I create code samples in e.g. Java, Python, ... or is this api explanation enough?

Reply:
I think the api is enough for now. Let's wait, if e.g. TV can work with it.